prow/entrypoint/run.go

```diff
@@ -36,6 +36,9 @@ const (
 	// InternalErrorCode is what we write to the marker file to
 	// indicate that we failed to start the wrapped command
 	InternalErrorCode = 127
+	// AbortedErrorCode is what we write to the marker file to
+	// indicate that we were terminated via a signal.
```

Comment:
Shouldn't this be `128+signal`?

Reply:
In this specific case we would need to behave the same for both signals.  So we either pass one exit code to mean abort or we check both.  Do we need to preserve the distinction between term and int?

Reply:
Depends on how well-behaved we want to be.

Reply:
If we think we’re the only reader of the marker file containing the status I’d say that the code is up to us.  We don’t actually signal that code back to the OS and kubelet